features/utils.py

```diff
@@ -113,15 +121,16 @@ def go_to_home_page(browser):
     home_page.flaw_list_exist()
 
 
-def go_to_specific_flaw_detail_page(browser, cve_id=TEST_FLAW_CVE_ID):
+def go_to_specific_flaw_detail_page(browser, embargoed=False):
     """
     Enter first flaw detail page which displayed in index page.
     """
+    cve_id = EMBARGOED_FLAW_CVE_ID if embargoed else PUBLIC_FLAW_CVE_ID
```

Comment:
I think we need more than two cve_id to do test, so it's better to keep the cve_id pamas.